src/focus-zone.ts

```diff
@@ -424,7 +424,11 @@ export function focusZone(container: HTMLElement, settings?: FocusZoneSettings):
 
     activeDescendantControl?.removeAttribute('aria-activedescendant')
     container.removeAttribute(hasActiveDescendantAttribute)
-    previouslyActiveElement?.removeAttribute(isActiveDescendantAttribute)
```

Comment:
What do you think of keeping this line as well, just in case the element isn't currently present in the DOM?

Reply:
Oh man, good call thank you 👍